packages/react-composites/src/composites/common/HiddenFocusStartPoint.tsx

```diff
@@ -12,13 +12,18 @@ import { useEffect, useState } from 'react';
  */
 export const HiddenFocusStartPoint = (): JSX.Element => {
   const [isMounted, setIsMounted] = useState(false);
+  // ariaHidden is used to hide the button from screen readers,
+  // we want to only focus it for once then never again
+  const [isUnfocused, setIsUnfocused] = useState(false);
+
   useEffect(() => {
     setIsMounted(true);
   }, []);
   return (
     <DefaultButton
       autoFocus
-      ariaHidden={true}
+      onBlur={() => setIsUnfocused(true)}
```

Comment:
Nice! I didn't onBlur was the about of onFocus. TIL!
![image](https://github.com/user-attachments/assets/69ed029c-4c26-45e2-994f-8a7865615a54)